Elements/src/extension.dib

```diff
@@ -0,0 +1,142 @@
+#!markdown
+
+This script is packaged with the NuGet package, and only
+called when the package is imported in a dotnet interactive
+notebook via:
+
+`#r "nuget: Hypar.Elements"`
+
+#!csharp
+
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using Microsoft.DotNet.Interactive;
+using Microsoft.DotNet.Interactive.Formatting;
+using Elements;
+using Elements.Geometry;
+using Elements.Serialization.glTF;
+
+var viewerSrc = @"
+<div id=""main_DIV_ID"" style=""height:400px;width:400px;""></div>
+</div>
+<script type=""module"">
+import * as THREE from 'https://unpkg.com/three@0.126.0/build/three.module.js';
+import { GLTFLoader } from 'https://unpkg.com/three@0.126.0/examples/jsm/loaders/GLTFLoader.js';
+import { OrbitControls } from 'https://unpkg.com/three@0.126.0/examples/jsm/controls/OrbitControls.js';
+const loader = new GLTFLoader();
+const scene = new THREE.Scene();
+
+let loaderScene = new THREE.Scene();
+scene.add(loaderScene);
+
+const width = WIDTH_VAR;
+const height = HEIGHT_VAR;
+const camera = new THREE.PerspectiveCamera(75, width / height, 0.1, 1000);
+const light = new THREE.AmbientLight(0xaaaaaa);
```

Comment:
I would like, if possible, to replicate the lighting/shading setup that's on Hypar currently to minimize surprises. I'm ok with this as v1.

Reply:
Will leave a TODO!